test/puppetlabs/puppetdb/cli/import_export_roundtrip_test.clj

```diff
@@ -15,24 +16,32 @@
             [puppetlabs.puppetdb.testutils.catalogs :as tuc]
             [puppetlabs.puppetdb.examples.reports :refer [reports]]
             [puppetlabs.puppetdb.testutils.reports :as tur]
+            [puppetlabs.puppetdb.testutils.facts :as tuf]
             [clj-time.core :refer [now]]
             [clj-time.coerce :refer [to-string]]
             [puppetlabs.puppetdb.utils :as utils]
             [puppetlabs.puppetdb.client :as pdb-client]
-            [puppetlabs.puppetdb.admin :as admin]
             [slingshot.slingshot :refer [throw+ try+]]
             [slingshot.test]
             [puppetlabs.puppetdb.testutils.services :as svc-utils]))
 
 (use-fixtures :each fixt/with-test-logging-silenced)
 
-(defn block-until-queue-empty
-  "Blocks the current thread until all messages from the queue have been processed."
-  []
-  (loop [depth (svc-utils/current-queue-depth)]
-    (when (< 0 depth)
-      (Thread/sleep 10)
-      (recur (svc-utils/current-queue-depth)))))
+(defn get-catalog [query-fn certname]
+  (->> ["=" "certname" certname]
+       (export/get-wireformatted-entity query-fn :catalogs)
+       first))
+
+(defn get-report [query-fn certname]
+  (->> ["=" "certname" certname]
+       (export/get-wireformatted-entity query-fn :reports)
+       first))
+
+(defn get-facts [query-fn certname]
+  (->> ["=" "certname" certname]
+       (export/get-wireformatted-entity query-fn :factsets)
+       first))
```

Comment:
I was thinking it could be nice to have all this in the `acceptance/` namespace and have a `testutils.clj` or `helpers.clj` there.